### ISSUE TITLE
replace `<` and `>` from message in the checkstyle format

### ISSUE THREAD

**Issue by lucasborin** [abaplint/abaplint#3147]
Given the following code snippet:
```abap
"! <p class="shorttext synchronized">My Class</p>
CLASS cl_my_class DEFINITION PUBLIC FINAL CREATE PUBLIC.
  PUBLIC SECTION.
    "! <p class="shorttext synchronized"></p>
    "!
    "! @parameter importing_a | <p class="shorttext synchronized"></p>
    "! @parameter importing_b | <p class="shorttext synchronized"></p>
    METHODS constructor IMPORTING importing_a TYPE REF TO if_my_interface_a
                                  importing_b TYPE REF TO if_my_interface_b OPTIONAL.

ENDCLASS.
```

When I run:
```
abaplint -f checkstyle
```

Then it creates an invalid xml:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<checkstyle version="8.36">
  <file name="./src/cl_cpe_erp_rounding.clas.abap">
    <error line="21" column="13" severity="Warning" message="Empty tag(s) in ABAP Doc for method constructor (&quot;! <P CLASS=&quot;SHORTTEXT SYNCHRONIZED&quot;></P>)" source="abaplint"/>
  </file>
</checkstyle>
```
```
The value of attribute "message" associated with an element type "error" must not contain the '<' character.
```
